### ISSUE TITLE
Paid subscriptions

### ISSUE THREAD

**Issue by pathumego** [fontman/fontman-desktop#2]
- What are our options and models? 
- How do we approach and pitch this to the user in-app experience?

**Comment by pathumego** [comment]
Can we use flattr API for subscriptions? Their Flattr Plus model is also interesting.

**Comment by lpsandaruwan** [comment]
Yes. We will introduce flattr based fund mechanism in Fontman version 1.0.0
